lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
+
+	/**
+	 * Gets the object containing definitions of groups and permissions in the system
+	 * @return PermissionsConfiguration
+	 */
+	public function getConfiguration();
+
+	/**
+	 * @param \User $user
+	 * @return string[] Get list of global and local groups to which the given user has been explicitly assigned to.
+	 */
+	public function getExplicitGroups(\User $user );
+
+	/**
+	 * @param \User $user
+	 * @return string[] Get list of global groups to which the given user has been explicitly assigned to.
+	 */
+	public function getExplicitGlobalGroups(\User $user );
+
+	/**
+	 * @param \User $user
+	 * @return string[] Get list of local groups to which the given user has been explicitly assigned to.
+	 */
+	public function getExplicitLocalGroups(\User $user );
+
+	/**
+	 * @param \User $user
+	 * @param $group string group name
+	 * @return bool True if user is member of the given group
+	 */
+	public function isInGroup(\User $user, $group );
+
+	/**
+	 * Get the list of explicit and implicit group memberships this user has.
+	 * This includes all explicit groups, plus 'user' if logged in,
+	 * '*' for all accounts, and autopromoted groups
+	 * @param \User $user
+	 * @param $reCacheAutomaticGroups
+	 * @return Array of String internal group names
```

Comment:
you've been using "string[]" somewhere above, can we keep one consistent way to specify that?